ob-sparql.el

```diff
@@ -64,10 +64,23 @@ to do that."
       (if (zerop (buffer-size))
           (error "error: URL '%s' is not accessible." endpoint-url)
         (goto-char (point-min))
-        (when (string-match "^.* 200 OK$" (thing-at-point 'line))
-          (search-forward "\n\n")
-          (delete-region (point-min) (point)))
-        (buffer-string)))))
+        (let ((result-is-csv (string-equal "text/csv" url-mime-accept-string))
+              (query-failed (not (string-match "^.* 200 OK$" (thing-at-point 'line)))))
+          (if query-failed
+              (buffer-string)
+            (search-forward "\n\n")
+            (delete-region (point-min) (point))
+            (org-babel-result-cond (cdr (assoc :result-params params))
+              (buffer-string)
+              (if result-is-csv
```

Comment:
Is this correct? It seems to me that if the the `url-mime-accept-string` is `test/csv` then it will always be turned into a table.